tests/lib/linter/linter.js

```diff
@@ -7442,7 +7472,13 @@ var a = "test2";
 
         it("should have file path passed to it", () => {
             const code = "/* this is code */";
-            const parseSpy = { parse: sinon.spy() };
+            const ast = {
+                tokens: [],
+                comments: [],
+                loc: {},
+                range: []
+            };
+            const parseSpy = { parse: sinon.fake.returns(ast) };
```

Comment:
```suggestion
            const parseSpy = { parse: sinon.spy(espree.parse) };
```

We could use `espree` here to return a valid AST.